cmd/joy/secret.go

```diff
@@ -44,6 +46,14 @@ func NewSecretSealCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "seal",
 		Short: "Encrypt secret",
+		Example: `  # Base usage with prompts
+  joy sealed-secret seal
+
+  # With pre-selected environment
+  joy sealed-secret seal -e staging
+
+  # Seal secret content piped in from stdin (required environment to be pre-selected)
+  cat ./secret.txt | joy sealed-secret seal -e production`,
```

Comment:
They say `cat` is overrated! ;)
```suggestion
joy sealed-secret seal -e production < ./secret.txt`,
```